src/controllers/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const home = require('./home');
+const login = require('./login');
+
+const router = express.Router();
+
+router.get('/', home.get);
+router.get('/admin/login', login.get);
```

Comment:
try to reduce  your code 
you can do this 
router
.get('/',home.get)
.get('/admin/login',login.get);